presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java

```diff
@@ -95,17 +83,19 @@ public void testInsertIntoValuesToHiveTableAllHiveSimpleTypes()
                         32767,
                         2147483647,
                         9223372036854775807L,
-                        123.34500122070312,
+                        123.345f,
                         234.567,
-                        Timestamp.valueOf("2015-05-10 12:15:35.123"),
+                        new BigDecimal("346"),
+                        new BigDecimal("345.67800"),
+                        parseTimestampInUTC("2015-05-10 12:15:35.123"),
                         Date.valueOf("2015-05-10"),
                         "ala ma kota",
                         "ala ma kot",
                         true,
                         "kot binarny".getBytes()));
     }
 
-    @Test(groups = {HIVE_CONNECTOR, QUARANTINE})
+    @Test(groups = {HIVE_CONNECTOR, SMOKE})
```

Comment:
there isn't a product test group "integration"; I'm not sure if smoke is used so much but I'll keep it as is